src/javascript/binary_japan/symbols.js

```diff
@@ -0,0 +1,20 @@
+if(typeof is_japan === 'function'){
+	Symbols._details = Symbols.details.bind({});
```

Comment:
Be careful with 'Symbols' name. We should reconsider the naming, since Symbol is a fundamental object in ES6 and we might mess stuff.